VirtoCommerce.CacheModule.Web/packages.config

```diff
@@ -5,17 +5,17 @@
   <package id="Common.Logging.Core" version="3.3.1" targetFramework="net461" />
   <package id="CommonServiceLocator" version="1.3" targetFramework="net461" />
   <package id="EntityFramework" version="6.2.0" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi" version="5.2.4" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.4" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.4" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.4" targetFramework="net461" />
   <package id="MSBuildTasks" version="1.5.0.235" targetFramework="net461" developmentDependency="true" />
   <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net461" />
   <package id="Unity" version="4.0.1" targetFramework="net461" />
   <package id="valueinjecter" version="2.3.3" targetFramework="net461" />
-  <package id="VirtoCommerce.Domain" version="2.25.3" targetFramework="net461" />
+  <package id="VirtoCommerce.Domain" version="2.25.10" targetFramework="net461" />
   <package id="VirtoCommerce.Module" version="0.17.0" targetFramework="net461" developmentDependency="true" />
-  <package id="VirtoCommerce.Platform.Core" version="2.13.28" targetFramework="net461" />
-  <package id="VirtoCommerce.Platform.Core.Web" version="2.12.5" targetFramework="net461" />
-  <package id="VirtoCommerce.Platform.Data" version="2.13.28" targetFramework="net461" />
+  <package id="VirtoCommerce.Platform.Core" version="2.13.35" targetFramework="net461" />
```

Comment:
You've updated the platform NuGet packages without changing the platform dependency version in the module manifest.